core/frontend/src/DisplayStyleState.ts

```diff
@@ -142,8 +142,14 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
   /** @internal */
   public get backgroundMapLayers(): MapLayerSettings[] { return this.settings.mapImagery.backgroundLayers; }
 
-  /** @internal */
-  public get backgroundMapBase(): BaseLayerSettings | undefined { return this.settings.mapImagery.backgroundBase; }
+  /** @beta */
+  public get backgroundMapBase(): BaseLayerSettings { return this.settings.mapImagery.backgroundBase; }
+
+  /** @beta */
+  public set backgroundMapBase(base: BaseLayerSettings) {
```

Comment:
@pmconne what about this?

Reply:
Fine for now, though I'm not a fan of how DisplayStyleState and Viewport have a bunch of properties that just return properties of their own properties (of their own properties...).
Ultimately we need to address the synchronization problem in MapImagerySettings and DisplayStyleSettings.